mapper_visitor/src/visitor.rs

```diff
@@ -0,0 +1,16 @@
+use chrono::{DateTime, FixedOffset};
+
+pub trait MeasurementVisitor {
+    type Error;
+
+    fn visit_measurement_type(&mut self, typename: &str) -> Result<(), Self::Error>;
```

Comment:
This measurement type was a mistake. It's required only for C8Y and has no counter part in ThinEdgeJson.

Reply:
The idea of the `Visitor` is to support all measurement formats, including C8Y. If you look at the `C8ySerializer` you see that we inject a `default_typename` and `default_timestamp`, which is used in case the input does not provide one.

If you'd write a `ThinEdgeSerializer`, the `visit_measurement_type` would just be a no-op.

Reply:
> in case the input does not provide one.

What I mean, is that the input will never provide one type name. Only C8Y requires such a name and we decided to provide always the same.

Reply:
The only place where this parameter makes sense is in the constructor of a `C8YJsonBuilder`.